src/docfx/template/TemplateSchema.cs

```diff
@@ -1,58 +1,18 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
-using System.IO;
-
 namespace Microsoft.Docs.Build
 {
     internal class TemplateSchema
     {
-        public string SchemaName { get; }
-
-        public bool IsPage { get; }
-
-        public JsonSchemaValidator JsonSchemaValidator { get; }
-
         public JsonSchema JsonSchema { get; }
 
-        public TemplateSchema(string schemaName, string schemaDir, string contentTemplateDir)
-        {
-            SchemaName = schemaName;
-            IsPage = GetIsPageCore(schemaName, contentTemplateDir);
-            (JsonSchema, JsonSchemaValidator) = GetJsonSchemaCore(schemaDir, schemaName);
-        }
-
-        private bool GetIsPageCore(string schemaName, string contentTemplateDir)
-        {
-            if (string.Equals(schemaName, "LandingData"))
-            {
-                return true;
-            }
-
-            return File.Exists(Path.Combine(contentTemplateDir, $"{schemaName}.html.primary.tmpl"))
-                || File.Exists(Path.Combine(contentTemplateDir, $"{schemaName}.html.primary.js"));
-        }
+        public JsonSchemaValidator JsonSchemaValidator { get; }
```

Comment:
nit: no need to keep this class? seems like tuple will simply do the job

Reply:
yep